src/x/instrument/config.go

```diff
@@ -85,6 +87,14 @@ func (mc *MetricsConfiguration) NewRootScope() (tally.Scope, io.Closer, error) {
 		if err != nil {
 			return nil, nil, err
 		}
+
+		// Turn off the default Prometheus process collector because it can be very
+		// slow if a lot of file descriptors are open.
+		toUnregister := promFork.NewProcessCollector(os.Getpid(), "")
```

Comment:
@robskillington I don't think this is the right way to do this / does not seem to work. Any thoughts on how I can do this without changing `uber-go/tally` or do I have no choice?